### DISCUSSION TITLE
Is there a way to selectively 'unfreeze' documents?

### DISCUSSION THREAD

**Discussion by danielvartan** [quarto-dev/quarto-cli#7540] [Q&A] [answered]
### Description

The `freeze: auto` feature is great, but you may have some documents that you always want to re-render.

```yaml
execute:
  freeze: auto [minus x.qmd]
```

## Example of application

I'm working with two profiles (HTML and PDF), and I need different content for `index.qmd` for each of them. To get around that, I created the following Quarto content.

````md
<!-- %:::% .common h1 begin %:::% -->
# Welcome {.unnumbered}
<!-- %:::% .common h1 end %:::% -->

```{r}
#| label: setup
#| echo: false
#| output: false

# library(here)

source(here::here("R/quarto-setup.R"))
```

::: {.content-visible when-format="html"}
{{< include qmd/_index-html.qmd >}}
:::

::: {.content-visible when-format="pdf"}
{{< include qmd/_index-pdf.qmd >}}
:::
````

Source: https://github.com/danielvartan/abnt/blob/main/index.qmd

As you can see, the index file does not change, but the content of `qmd/_index-html.qmd` or ` qmd/_index-pdf.qmd` can. If `index.qmd` could be unfrozen, it would render any changes in those two files.

**Comment by mcanouil** [comment, marked as answer]
Set `freeze: false` in the documents you are talking about.

Side note: the proper option to not include a code cell is `include: false` (not echo+output).

Also:
- https://github.com/quarto-dev/quarto-cli/issues/6793
- https://github.com/quarto-dev/quarto-cli/issues/2795

**Reply by danielvartan** [reply]
Thank you @mcanouil ! ❤️
